spec/graphql/mutations/repository/create_repository_mutation_spec.rb

```diff
@@ -72,4 +72,14 @@
         to include(include('message' => 'name is already taken'))
     end
   end
+
+  context 'Not signed in' do
+    let(:context) { {} }
+
+    it 'returns an error' do
+      expect(subject['data']['createRepository']).to be(nil)
+      expect(subject['errors']).
+        to include(include('message' => "You're not authorized to do this"))
+    end
+  end
```

Comment:
This is the only case where I would like to have another example testing if you are authorised to perform the action if you are signed in as a "bad user" (not the owner). It would be checking if the resource `owner` passed to `authorize!` is the correct one.

https://github.com/ontohub/ontohub-backend/blob/f9f92395b957e908572cf8b893c51972b5f3d9b4/app/graphql/mutations/repository/create_repository_mutation.rb#L13-L19

Reply:
Done in 3432f19